platform-api/src/main/java/io/datacater/core/kubernetes/DataCaterK8sConfig.java

```diff
@@ -48,4 +51,42 @@ private DataCaterK8sConfig() {}
       ConfigProvider.getConfig()
           .getOptionalValue("datacater.pythonrunner.replicas", Integer.class)
           .orElse(1);
+
+  private static final String CPU = "cpu";
+  private static final String MEMORY = "memory";
+  static final String PYTHONRUNNER_RESOURCES_REQUESTS_CPU =
+      ConfigProvider.getConfig()
+          .getOptionalValue("datacater.pythonrunner.resources.requests.cpu", String.class)
+          .orElse("0.1");
+  static final String PYTHONRUNNER_RESOURCES_REQUESTS_MEMORY =
+      ConfigProvider.getConfig()
+          .getOptionalValue("datacater.pythonrunner.resources.requests.memory", String.class)
+          .orElse("200Mi");
```

Comment:
`200Mi` feels too much. In my Minikube cluster, I observed a memory usage of ~ `95Mi` when not using any requests/limits.